Timestamp,Output
2025-02-02 21:33:03,Traceback (most recent call last):
2025-02-02 21:33:03,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-02 21:33:03,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-02 21:33:03,ModuleNotFoundError: No module named 'transformers'
2025-02-02 21:33:40,Traceback (most recent call last):
2025-02-02 21:33:40,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-02 21:33:40,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-02 21:33:40,ModuleNotFoundError: No module named 'transformers'
2025-02-02 21:34:16,Traceback (most recent call last):
2025-02-02 21:34:16,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-02 21:34:16,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-02 21:34:16,ModuleNotFoundError: No module named 'transformers'
2025-02-02 21:34:33,Traceback (most recent call last):
2025-02-02 21:34:33,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-02 21:34:33,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-02 21:34:33,ModuleNotFoundError: No module named 'transformers'
2025-02-02 21:34:33,Traceback (most recent call last):
2025-02-02 21:34:33,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-02 21:34:33,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-02 21:34:33,ModuleNotFoundError: No module named 'transformers'
2025-02-02 21:34:36,Traceback (most recent call last):
2025-02-02 21:34:36,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-02 21:34:36,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-02 21:34:36,ModuleNotFoundError: No module named 'transformers'
2025-02-02 21:35:10,Traceback (most recent call last):
2025-02-02 21:35:10,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-02 21:35:10,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-02 21:35:10,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:41:28,Traceback (most recent call last):
2025-02-03 11:41:28,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:41:28,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:41:28,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:42:05,Traceback (most recent call last):
2025-02-03 11:42:05,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:42:05,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:42:05,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:42:42,Traceback (most recent call last):
2025-02-03 11:42:42,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:42:42,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:42:42,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:43:00,Traceback (most recent call last):
2025-02-03 11:43:00,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:43:00,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:43:00,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:43:00,Traceback (most recent call last):
2025-02-03 11:43:00,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:43:00,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:43:00,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:43:03,Traceback (most recent call last):
2025-02-03 11:43:03,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:43:03,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:43:03,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:43:39,Traceback (most recent call last):
2025-02-03 11:43:39,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:43:39,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:43:39,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:45:34,Traceback (most recent call last):
2025-02-03 11:45:34,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:45:34,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:45:34,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:46:20,Traceback (most recent call last):
2025-02-03 11:46:20,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:46:20,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:46:20,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:46:41,Traceback (most recent call last):
2025-02-03 11:46:41,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:46:41,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:46:41,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:46:41,Traceback (most recent call last):
2025-02-03 11:46:41,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:46:41,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:46:41,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:47:01,Traceback (most recent call last):
2025-02-03 11:47:01,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:47:01,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:47:01,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:47:47,Traceback (most recent call last):
2025-02-03 11:47:47,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:47:47,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:47:47,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:48:31,Traceback (most recent call last):
2025-02-03 11:48:31,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:48:31,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:48:31,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:48:52,Traceback (most recent call last):
2025-02-03 11:48:52,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:48:52,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:48:52,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:48:52,Traceback (most recent call last):
2025-02-03 11:48:52,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:48:52,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:48:52,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:49:12,Traceback (most recent call last):
2025-02-03 11:49:12,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:49:12,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:49:12,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:49:33,Traceback (most recent call last):
2025-02-03 11:49:33,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:49:33,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:49:33,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:49:33,Traceback (most recent call last):
2025-02-03 11:49:33,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:49:33,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:49:33,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:49:53,Traceback (most recent call last):
2025-02-03 11:49:53,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:49:53,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:49:53,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:49:53,Traceback (most recent call last):
2025-02-03 11:49:53,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:49:53,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:49:53,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:49:53,Traceback (most recent call last):
2025-02-03 11:49:53,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:49:53,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:49:53,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:50:11,Traceback (most recent call last):
2025-02-03 11:50:11,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:50:11,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:50:11,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:50:11,Traceback (most recent call last):
2025-02-03 11:50:11,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:50:11,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:50:11,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:50:30,Traceback (most recent call last):
2025-02-03 11:50:30,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:50:30,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:50:30,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:50:51,Traceback (most recent call last):
2025-02-03 11:50:51,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:50:51,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:50:51,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:50:51,Traceback (most recent call last):
2025-02-03 11:50:51,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:50:51,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:50:51,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:51:10,Traceback (most recent call last):
2025-02-03 11:51:10,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:51:10,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:51:10,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:51:54,Traceback (most recent call last):
2025-02-03 11:51:54,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:51:54,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:51:54,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:52:38,Traceback (most recent call last):
2025-02-03 11:52:38,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:52:38,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:52:38,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:52:59,Traceback (most recent call last):
2025-02-03 11:52:59,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:52:59,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:52:59,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:52:59,Traceback (most recent call last):
2025-02-03 11:52:59,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:52:59,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:52:59,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:53:18,Traceback (most recent call last):
2025-02-03 11:53:18,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:53:18,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:53:18,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:54:01,Traceback (most recent call last):
2025-02-03 11:54:01,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:54:01,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:54:01,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:54:43,Traceback (most recent call last):
2025-02-03 11:54:43,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:54:43,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:54:43,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:55:05,Traceback (most recent call last):
2025-02-03 11:55:05,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:55:05,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:55:05,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:55:05,Traceback (most recent call last):
2025-02-03 11:55:05,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:55:05,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:55:05,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:55:25,Traceback (most recent call last):
2025-02-03 11:55:25,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:55:25,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:55:25,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:55:46,Traceback (most recent call last):
2025-02-03 11:55:46,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:55:46,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:55:46,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:55:46,Traceback (most recent call last):
2025-02-03 11:55:46,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:55:46,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:55:46,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:56:05,Traceback (most recent call last):
2025-02-03 11:56:05,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:56:05,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:56:05,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:56:05,Traceback (most recent call last):
2025-02-03 11:56:05,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:56:05,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:56:05,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:56:05,Traceback (most recent call last):
2025-02-03 11:56:05,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:56:05,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:56:05,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:56:23,Traceback (most recent call last):
2025-02-03 11:56:23,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:56:23,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:56:23,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:56:23,Traceback (most recent call last):
2025-02-03 11:56:23,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:56:23,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:56:23,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:56:42,Traceback (most recent call last):
2025-02-03 11:56:42,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:56:42,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:56:42,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:57:03,Traceback (most recent call last):
2025-02-03 11:57:03,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:57:03,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:57:03,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:57:03,Traceback (most recent call last):
2025-02-03 11:57:03,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:57:03,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:57:03,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:57:22,Traceback (most recent call last):
2025-02-03 11:57:22,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:57:22,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:57:22,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:57:42,Traceback (most recent call last):
2025-02-03 11:57:42,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:57:42,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:57:42,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:57:42,Traceback (most recent call last):
2025-02-03 11:57:42,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:57:42,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:57:42,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:01,Traceback (most recent call last):
2025-02-03 11:58:01,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:01,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:01,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:01,Traceback (most recent call last):
2025-02-03 11:58:01,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:01,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:01,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:01,Traceback (most recent call last):
2025-02-03 11:58:01,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:01,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:01,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:19,Traceback (most recent call last):
2025-02-03 11:58:19,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:19,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:19,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:19,Traceback (most recent call last):
2025-02-03 11:58:19,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:19,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:19,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:38,Traceback (most recent call last):
2025-02-03 11:58:38,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:38,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:38,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:38,Traceback (most recent call last):
2025-02-03 11:58:38,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:38,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:38,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:38,Traceback (most recent call last):
2025-02-03 11:58:38,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:38,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:38,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:56,Traceback (most recent call last):
2025-02-03 11:58:56,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:56,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:56,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:56,Traceback (most recent call last):
2025-02-03 11:58:56,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:56,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:56,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:56,Traceback (most recent call last):
2025-02-03 11:58:56,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:56,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:56,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:56,Traceback (most recent call last):
2025-02-03 11:58:56,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:56,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:56,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:59,Traceback (most recent call last):
2025-02-03 11:58:59,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:59,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:59,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:59,Traceback (most recent call last):
2025-02-03 11:58:59,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:59,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:59,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:58:59,Traceback (most recent call last):
2025-02-03 11:58:59,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:58:59,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:58:59,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:59:15,Traceback (most recent call last):
2025-02-03 11:59:15,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:59:15,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:59:15,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:59:15,Traceback (most recent call last):
2025-02-03 11:59:15,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:59:15,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:59:15,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:59:34,Traceback (most recent call last):
2025-02-03 11:59:34,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:59:34,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:59:34,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:59:34,Traceback (most recent call last):
2025-02-03 11:59:34,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:59:34,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:59:34,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:59:34,Traceback (most recent call last):
2025-02-03 11:59:34,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:59:34,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:59:34,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:59:50,Traceback (most recent call last):
2025-02-03 11:59:50,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:59:50,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:59:50,ModuleNotFoundError: No module named 'transformers'
2025-02-03 11:59:51,Traceback (most recent call last):
2025-02-03 11:59:51,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 11:59:51,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 11:59:51,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:00:09,Traceback (most recent call last):
2025-02-03 12:00:09,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:00:09,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:00:09,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:00:30,Traceback (most recent call last):
2025-02-03 12:00:30,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:00:30,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:00:30,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:00:30,Traceback (most recent call last):
2025-02-03 12:00:30,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:00:30,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:00:30,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:00:49,Traceback (most recent call last):
2025-02-03 12:00:49,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:00:49,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:00:49,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:01:09,Traceback (most recent call last):
2025-02-03 12:01:09,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:01:09,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:01:09,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:01:09,Traceback (most recent call last):
2025-02-03 12:01:09,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:01:09,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:01:09,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:01:28,Traceback (most recent call last):
2025-02-03 12:01:28,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:01:28,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:01:28,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:01:28,Traceback (most recent call last):
2025-02-03 12:01:28,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:01:28,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:01:28,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:01:28,Traceback (most recent call last):
2025-02-03 12:01:28,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:01:28,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:01:28,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:01:44,Traceback (most recent call last):
2025-02-03 12:01:44,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:01:44,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:01:44,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:01:44,Traceback (most recent call last):
2025-02-03 12:01:44,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:01:44,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:01:44,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:02:02,Traceback (most recent call last):
2025-02-03 12:02:02,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:02:02,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:02:02,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:02:22,Traceback (most recent call last):
2025-02-03 12:02:22,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:02:22,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:02:22,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:02:22,Traceback (most recent call last):
2025-02-03 12:02:22,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:02:22,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:02:22,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:02:40,Traceback (most recent call last):
2025-02-03 12:02:40,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:02:40,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:02:40,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:03:24,Traceback (most recent call last):
2025-02-03 12:03:24,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:03:24,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:03:24,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:04:07,Traceback (most recent call last):
2025-02-03 12:04:07,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:04:07,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:04:07,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:04:28,Traceback (most recent call last):
2025-02-03 12:04:28,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:04:28,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:04:28,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:04:28,Traceback (most recent call last):
2025-02-03 12:04:28,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:04:28,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:04:28,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:04:47,Traceback (most recent call last):
2025-02-03 12:04:47,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:04:47,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:04:47,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:05:30,Traceback (most recent call last):
2025-02-03 12:05:30,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:05:30,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:05:30,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:06:12,Traceback (most recent call last):
2025-02-03 12:06:12,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:06:12,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:06:12,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:06:33,Traceback (most recent call last):
2025-02-03 12:06:33,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:06:33,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:06:33,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:06:33,Traceback (most recent call last):
2025-02-03 12:06:33,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:06:33,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:06:33,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:06:51,Traceback (most recent call last):
2025-02-03 12:06:51,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:06:51,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:06:51,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:07:11,Traceback (most recent call last):
2025-02-03 12:07:11,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:07:11,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:07:11,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:07:11,Traceback (most recent call last):
2025-02-03 12:07:11,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:07:11,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:07:11,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:07:30,Traceback (most recent call last):
2025-02-03 12:07:30,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:07:30,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:07:30,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:07:30,Traceback (most recent call last):
2025-02-03 12:07:30,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:07:30,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:07:30,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:07:30,Traceback (most recent call last):
2025-02-03 12:07:30,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:07:30,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:07:30,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:07:47,Traceback (most recent call last):
2025-02-03 12:07:47,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:07:47,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:07:47,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:07:47,Traceback (most recent call last):
2025-02-03 12:07:47,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:07:47,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:07:47,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:08:04,Traceback (most recent call last):
2025-02-03 12:08:04,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:08:04,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:08:04,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:08:24,Traceback (most recent call last):
2025-02-03 12:08:24,Traceback (most recent call last):
2025-02-03 12:08:24,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:08:24,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:08:24,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:08:24,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:08:24,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:08:24,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:08:42,Traceback (most recent call last):
2025-02-03 12:08:42,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:08:42,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:08:42,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:09:25,Traceback (most recent call last):
2025-02-03 12:09:25,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:09:25,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:09:25,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:10:08,Traceback (most recent call last):
2025-02-03 12:10:08,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:10:08,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:10:08,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:10:28,Traceback (most recent call last):
2025-02-03 12:10:28,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:10:28,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:10:28,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:10:28,Traceback (most recent call last):
2025-02-03 12:10:28,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:10:28,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:10:28,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:10:46,Traceback (most recent call last):
2025-02-03 12:10:46,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:10:46,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:10:46,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:11:29,Traceback (most recent call last):
2025-02-03 12:11:29,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:11:29,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:11:29,ModuleNotFoundError: No module named 'transformers'
2025-02-03 12:12:38,Traceback (most recent call last):
2025-02-03 12:12:38,"File ""/home/user/Desktop/CODE/Adit/speech.py"", line 93, in <module>"
2025-02-03 12:12:38,"from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor"
2025-02-03 12:12:38,ModuleNotFoundError: No module named 'transformers'
